lib/bike_brigade/messaging/slack_webhook.ex

```diff
@@ -40,6 +40,23 @@ defmodule BikeBrigade.Messaging.SlackWebhook do
     :ok = SlackApi.send_webook(payload)
   end
 
+  def post_message(message) do
```

Comment:
I think we're still needing to manage two webhook URLs in the config. Let me know if you want a 🍐 on this

Reply:
Ya I haven't touched this since we chatted on tuesday. I want to also attempt the refactor so that the rider is created and we have the `id` to pass to slack.

Reply:
I am going to tackle the slack refactor in a separate PR. Discussed earlier (leaving here for posterity), we are currently using the [`Incoming Webhooks`](https://api.slack.com/messaging/webhooks) API. This has a disadvantage in that each app installed in each channel has its own (private) URL. Meaning we'd have an `.env` file that looks something like

```
channel_a="some-secret-url"
channel_b="another-secret-url"
channel_c="yet-another-secret-url"
```

If we use [`chat.postMessage`](https://api.slack.com/methods/chat.postMessage), we can use a single auth token stored in the `.env` file, and commit the `channel_ids` to the repo as they are not sensitive.